commit ce4f015978d250718ec8dd9f50721da90a9bd18b
Author: Louis Sautier <sautier.louis@gmail.com>
Date:   Mon Aug 20 12:42:29 2018 +0200

    Skip int() keyword tests with Python 3.7
    
    The first argument must be positional since
    https://github.com/python/cpython/commit/2e5642422f6234fd8d0c082142b27340e588f96e

diff --git a/tests/test_future/test_int.py b/tests/test_future/test_int.py
index f1d9c5d..e306550 100644
--- a/tests/test_future/test_int.py
+++ b/tests/test_future/test_int.py
@@ -265,6 +265,9 @@ class IntTestCases(unittest.TestCase):
     def test_no_args(self):
         self.assertEqual(int(), 0)
 
+    @unittest.skipIf(sys.version_info >= (3, 7),
+            "The first parameter must be positional with Python >= 3.7"
+    )
     def test_keyword_args(self):
         # Test invoking int() using keyword arguments.
         self.assertEqual(int(x=1.2), 1)

commit d7a3451753982d944ae0894c9e9f01d0460d03d5
Author: Louis Sautier <sautier.louis@gmail.com>
Date:   Mon Aug 20 11:17:59 2018 +0200

    Make test_bad_status_repr work for Python 3.7

diff --git a/tests/test_future/test_httplib.py b/tests/test_future/test_httplib.py
index 968e933..a1e2b19 100644
--- a/tests/test_future/test_httplib.py
+++ b/tests/test_future/test_httplib.py
@@ -197,7 +197,7 @@ class BasicTest(TestCase):
         if not utils.PY3:
             self.assertEqual(repr(exc), '''BadStatusLine("u\'\'",)''')
         else:
-            self.assertEqual(repr(exc), '''BadStatusLine("\'\'",)''')
+            self.assertIn(repr(exc), ('''BadStatusLine("''",)''', '''BadStatusLine("''")'''))
 
     def test_partial_reads(self):
         # if we have a length, the system knows when to close itself
